<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Register</class>
 <widget class="QMainWindow" name="Register">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>240</width>
    <height>320</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLineEdit" name="nameField">
      <property name="placeholderText">
       <string>enter name</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="usernameField">
      <property name="placeholderText">
       <string>enter username</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="passwordField">
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
      <property name="placeholderText">
       <string>enter password</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="emailField">
      <property name="placeholderText">
       <string>enter email</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="registerb">
      <property name="text">
       <string>register</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>240</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
